ci-operator/step-registry/stackrox/e2e/stackrox-e2e-ref.yaml

```diff
@@ -23,6 +23,8 @@ ref:
     default: ''
   - name: COLLECTION_METHOD
     default: ebpf
+  - name: WORKER_NODE_COUNT
+    default: '3'
```

Comment:
Doesn't make sense here to me. This will be set for all cluster flavors but have no affect if changed for e.g. gke, aro, eks, etc
```suggestion
```

Reply:
IIRC the prow config check rejected my `WORKER_NODE_COUNT: 4` overrides in the other files unless I declared this variable here. I'm happy to do this differently but I don't know how :sob:

Reply:
Ok, I can imagine it might have an issue... What about what we do to get an extra node for qa_e2e tests for OCP: https://github.com/stackrox/stackrox/blob/4f295452773a55f5d65e6ccf72a785ce58fe7e50/scripts/ci/jobs/ocp_qa_e2e_tests.py#L18

Reply:
This looks more straightforward, but is probably slower... do you know how long scaling takes?
OTOH the provisioning and scaling might still fit within the build time... :thinking:

Reply:
Ah, it's actually faster than I thought:

```
**** 12:34:34: About to provision
[...]
**** 12:35:23: provisioned
```